src/views/login/login.view.jsx

```diff
@@ -18,12 +18,12 @@ import CreateAccountAuth from './components/create-account-auth/create-account-a
 import Button from '../shared/button/button.view'
 import { LOAD_ETHEREUM_NETWORK_ERROR } from '../../store/global/global.reducer'
 import ChainIdError from './components/chain-id-error/chain-id-error.view'
-import MetaMaskError from './components/metamask-error/metamask-error.view'
 import { PRIVACY_POLICY_URL, TERMS_OF_SERVICE_URL } from '../../constants'
 import UnderMaintenanceError from './components/under-maintenance-error/under-maintenance-error.view'
 
 export const WalletName = {
   METAMASK: 'metaMask',
+  WALLET_CONNECT: 'wallet connect',
```

Comment:
I think that it would be better to use `walletConnect` without spaces as the value here